Sources/SwiftSyntaxBuilder/gyb_generated/Tokens.swift

```diff
@@ -19,178 +19,340 @@ enum Tokens {
+  /// The `(` keyword
+  static let `leftParen` = SyntaxFactory.makeLeftParenKeyword()
+  /// The `)` keyword
+  static let `rightParen` = SyntaxFactory.makeRightParenKeyword()
+  /// The `{` keyword
+  static let `leftBrace` = SyntaxFactory.makeLeftBraceKeyword()
+    .withLeadingTrivia(.spaces(1))
+  /// The `}` keyword
+  static let `rightBrace` = SyntaxFactory.makeRightBraceKeyword()
+    .withLeadingTrivia(.spaces(1))
+  /// The `[` keyword
+  static let `leftSquareBracket` = SyntaxFactory.makeLeftSquareBracketKeyword()
+  /// The `]` keyword
+  static let `rightSquareBracket` = SyntaxFactory.makeRightSquareBracketKeyword()
+  /// The `<` keyword
+  static let `leftAngle` = SyntaxFactory.makeLeftAngleKeyword()
+    .withLeadingTrivia(.spaces(1))
+    .withTrailingTrivia(.spaces(1))
+  /// The `>` keyword
+  static let `rightAngle` = SyntaxFactory.makeRightAngleKeyword()
+    .withLeadingTrivia(.spaces(1))
+    .withTrailingTrivia(.spaces(1))
+  /// The `.` keyword
+  static let `period` = SyntaxFactory.makePeriodKeyword()
+  /// The `.` keyword
+  static let `prefixPeriod` = SyntaxFactory.makePrefixPeriodKeyword()
+  /// The `,` keyword
+  static let `comma` = SyntaxFactory.makeCommaKeyword()
+    .withLeadingTrivia(.spaces(1))
+  /// The `...` keyword
+  static let `ellipsis` = SyntaxFactory.makeEllipsisKeyword()
+  /// The `:` keyword
+  static let `colon` = SyntaxFactory.makeColonKeyword()
+    .withLeadingTrivia(.spaces(1))
+  /// The `;` keyword
+  static let `semicolon` = SyntaxFactory.makeSemicolonKeyword()
+  /// The `=` keyword
+  static let `equal` = SyntaxFactory.makeEqualKeyword()
+    .withLeadingTrivia(.spaces(1))
+    .withTrailingTrivia(.spaces(1))
+  /// The `@` keyword
+  static let `atSign` = SyntaxFactory.makeAtSignKeyword()
+  /// The `#` keyword
+  static let `pound` = SyntaxFactory.makePoundKeyword()
+  /// The `&` keyword
+  static let `prefixAmpersand` = SyntaxFactory.makePrefixAmpersandKeyword()
+    .withLeadingTrivia(.spaces(1))
+    .withTrailingTrivia(.spaces(1))
+  /// The `->` keyword
+  static let `arrow` = SyntaxFactory.makeArrowKeyword()
+    .withLeadingTrivia(.spaces(1))
+  /// The ``` keyword
+  static let `backtick` = SyntaxFactory.makeBacktickKeyword()
+  /// The `\\` keyword
+  static let `backslash` = SyntaxFactory.makeBackslashKeyword()
+  /// The `!` keyword
+  static let `exclamationMark` = SyntaxFactory.makeExclamationMarkKeyword()
+  /// The `?` keyword
+  static let `postfixQuestionMark` = SyntaxFactory.makePostfixQuestionMarkKeyword()
+  /// The `?` keyword
+  static let `infixQuestionMark` = SyntaxFactory.makeInfixQuestionMarkKeyword()
+  /// The `\"` keyword
+  static let `stringQuote` = SyntaxFactory.makeStringQuoteKeyword()
+  /// The `\'` keyword
+  static let `singleQuote` = SyntaxFactory.makeSingleQuoteKeyword()
+  /// The `\"\"\"` keyword
+  static let `multilineStringQuote` = SyntaxFactory.makeMultilineStringQuoteKeyword()
```

Comment:
It’s a bit unfortunate that we over-escape the text in `Token.py` so it ends up `\`-escaped here.

The simplest solution I found on the internet would be print it as
```python
token.text.encode('utf-8').decode('unicode_escape')
```

Not sure if that breaks any other comments though. If it does and you can’t find an easy solution to remove the `\` here, just leave it as is for now.